### PR TITLE
links; present tense; typo

### PR DESCRIPTION
**Pull request by nikojpapa**
This puts non-scholarly and non-essential links into comments so that they will not appear in markdown rendering.

This also removes future tense from the EIP and fixes a typo.

### CODE REVIEW

**Comment by nikojpapa** [comment]
I would like to keep links in, if possible, so that I can use them as reference myself, and I'm sure others would find them useful as well. I've put them in reference style links, though, but never reference them, so that they are hidden and do not appear as links when the markdown is rendered. Do you think this would be acceptable as an alternative to removing them completely, @djrtwo ?

**Comment by djrtwo** [comment]
Maybe! that is a reasonable idea to present. I'm unsure what the editors will think.

@lightclient is an editor. Maybe he can way in the conundrum of this being pretty academically rooted and some links (e.g. academic papers) being pretty important in the justification

File: EIPS/eip-X.md [hunk 1]
```diff
@@ -25,49 +26,60 @@ signature verification schemes.
...
-- Providing solutions MUST follow a [commit-reveal](https://medium.com/swlh/exploring-commit-reveal-schemes-on-ethereum-c4ff5a777db8) scheme to prevent [front running](https://solidity-by-example.org/hacks/front-running/.).
+- Providing solutions MUST follow a commit-reveal scheme to prevent front running.
 - This scheme MUST require one day between commit and reveal.
 
+[commit-reveal Link]: https://medium.com/swlh/exploring-commit-reveal-schemes-on-ethereum-c4ff5a777db8
+[front running Link]: https://solidity-by-example.org/hacks/front-running/
+
 ### Bounty funds
 
 - Funds of at least 50,000 gas SHALL be sent to the contract as a bounty.
```
Comment (lightclient) [inline EIPS/eip-X.md:85]: Not super clear on this point. Gas and balance are different units. You should specify the minimum bounty size in terms of balance (amount of wei, gwei, etc). The minimum gas price on mainnet is 7 wei post 1559, but you should assume mainnet is closer to 20 gwei. So 20 gwei * 50,000 would roughly the minimum bounty you would want to provide.

**Comment by lightclient** [comment]
This looks really nice overall. I think the bot is going to give you a pretty hard time with the links, but you will certainly be able to include most of them. I would check out EIP-1 to see the latest on what links are acceptable: https://eips.ethereum.org/EIPS/eip-1#linking-to-external-resources

File: EIPS/eip-X.md (hunk 1 above)
Comment (nikojpapa) [reply EIPS/eip-X.md:85]: The issue is if the price increases, the bounty should be increased as well. The bounty should cover 50,000 gas at all times.

**Pull request merged**